R/nca_linear.R

```diff
@@ -0,0 +1,19 @@
+#' NCA core function to perform linear trapezoid calculations for post-infusion
+#'
+#' @inheritParams nca_trapezoid
+#'
+nca_linear <- function(data) {
+  sum(diff(data$time) * (mean_step(data$dv)))
+}
+
+#' Calculate mean between two values in a vector
+#' Used in NCA for calculation of mean between two or more timepoints.
+#' 
+#' @x vector of numeric values
```

Comment:
need to update the roxygen syntax and then re-run `devtools::document()`

```suggestion
#' @param x vector of numeric values
```